management/server/http/api/openapi.yml

```diff
@@ -578,8 +649,7 @@ components:
                   "account.create", "account.setting.peer.login.expiration.update", "account.setting.peer.login.expiration.disable", "account.setting.peer.login.expiration.enable",
                   "route.add", "route.delete", "route.update",
                   "nameserver.group.add", "nameserver.group.delete", "nameserver.group.update",
-                  "peer.ssh.disable", "peer.ssh.enable", "peer.rename", "peer.login.expiration.disable", "peer.login.expiration.enable"
-          ]
+                  "peer.ssh.disable", "peer.ssh.enable", "peer.rename", "peer.login.expiration.disable", "peer.login.expiration.enable" ]
```

Comment:
we need to add corresponding policy events here.
And in the dashboard: https://github.com/netbirdio/dashboard/blob/bce75c1ca969def657dbd079dbe49970f7ef1500/src/views/Activity.tsx#L132
https://github.com/netbirdio/dashboard/blob/bce75c1ca969def657dbd079dbe49970f7ef1500/src/views/Activity.tsx#L76

Reply:
Done https://github.com/netbirdio/dashboard/pull/147